src/chargen.cpp

```diff
@@ -916,6 +949,13 @@ int parse_assign(struct descriptor_data *d, const char *arg)
   if (d->ccr.pr[lowest_kosher_magic_slot] == PR_RACE)
     lowest_kosher_magic_slot--;
 
+  // Except for otaku who are always mundane lol
+  if (*arg == 2 && d->ccr.is_otaku && d->ccr.temp <= 1) {

```

Comment:
What's the function of `d->ccr.temp` here? Is there ever a situation in which an otaku would be able to set magic?

Reply:
Artifact of copy+pasting code. Deleted!